actioncable/lib/action_cable/test_helper.rb

```diff
@@ -0,0 +1,124 @@
+module ActionCable
+  # Provides helper methods for testing Action Cable broadcasting
+  module TestHelper
+    extend ActiveSupport::Concern
+
+    included do
+      def before_setup # :nodoc:
+        server = ActionCable.server
+        test_adapter = ActionCable::SubscriptionAdapter::Test.new(server)
+
+        @old_pubsub_adapter = server.pubsub
+
+        server.instance_variable_set(:@pubsub, test_adapter)
+        super
+      end
+
+      def after_teardown # :nodoc:
+        super
+        ActionCable.server.instance_variable_set(:@pubsub, @old_pubsub_adapter)
+      end
```

Comment:
Why is this setup/teardown needed when we're setting the test adapter in cable.yml?